src/store/slices/AppNavSlice.ts

```diff
@@ -0,0 +1,21 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit'
```

Comment:
Is this Slice used?

Reply:
Yeah I was using this slice to hide the nav bar and show notifications but ended up with a different approach. Will remove